src/runtime/meter-toast-api.js

```diff
@@ -68,12 +68,25 @@ export class MeterToastApi {
     this.deps_
       .callbacks()
       .triggerFlowStarted(SubscriptionFlows.SHOW_METER_TOAST);
-    this.win_.addEventListener('click', this.sendCloseRequestFunction_);
     this.activityIframeView_.on(
       ViewSubscriptionsResponse,
       this.startNativeFlow_.bind(this)
     );
-    return this.dialogManager_.openView(this.activityIframeView_);
+    return this.dialogManager_
+      .openView(this.activityIframeView_)
+      .then((_unusedDialog) => {
+        // Allow closing of the iframe with any scroll or click event.
+        this.win_.addEventListener('click', this.sendCloseRequestFunction_);
+        this.win_.addEventListener(
+          'touchstart',
+          this.sendCloseRequestFunction_
+        );
+        this.win_.addEventListener('mousedown', this.sendCloseRequestFunction_);
+        this.win_.addEventListener('wheel', this.sendCloseRequestFunction_);
+        // Making body 'hidden' to prevent scrolling while swiping on the iframe.
```

Comment:
Could you update the comment to specify the body's "overflow" property is being set to "hidden", not the body itself?

Reply:
Done